commands/account/remove.ts

```diff
@@ -50,26 +62,28 @@ exports.handler = async options => {
   );
 
   // Get updated version of the config
-  loadConfig(getConfigPath(), options);
+  loadConfig(getConfigPath()!, args as CLIOptions);
```

Comment:
I think in this case you can probably just omit `args` from the `loadConfig` call - it looks like the fields that LDL are looking for in `CLIOptions` will never exist in this command

Reply:
Nice catch 🙏